### PR TITLE
Clarify language regarding the sidecar webhook.

### LINKED ISSUE #3591: sidecar injection understand what happened
page is confusing:  https://preliminary.istio.io/docs/setup/kubernetes/additional-setup/sidecar-injection/#understanding-what-happened

it mentioned about `The sidecar injector pod(s) should be restarted after modifying the mutatingwebhookconfiguration` however, it didn't mention what/why someone needs to modify the mutatingwebhookconfiguration.

### PR DESCRIPTION
**Pull request by rcaballeromx**
Fixes #3591 

Signed-off-by: rcaballeromx <grca@google.com>

### CODE REVIEW

**Comment by rcaballeromx** [comment]
@ostromart PTAL

File: content/docs/setup/kubernetes/additional-setup/sidecar-injection/index.md [hunk 1]
```diff
@@ -160,20 +160,33 @@ sleep-776b7bcdcd-gmvnr   1/1       Running       0          2s
...
-[admissionregistration.k8s.io/v1beta1#MutatingWebhookConfiguration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#mutatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io)
-configures when the webhook is invoked by Kubernetes. The default
-supplied with Istio selects pods in namespaces with label
-`istio-injection=enabled`.  The set of namespaces in which injection
-is applied can be changed by editing the `MutatingWebhookConfiguration`
-with `kubectl edit mutatingwebhookconfiguration
-istio-sidecar-injector`.
+When Kubernetes invokes the webhook, the [admissionregistration.k8s.io/v1beta1#MutatingWebhookConfiguration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#mutatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io)
+
+configuration is applied. The default configuration labels the pods in the
```
Comment (ostromart) [inline content/docs/setup/kubernetes/additional-setup/sidecar-injection/index.md]: The default configuration injects the sidecar into pods in any namespace with a `istio-injection=enabled` label. The istio-sidecar-injector ConfigMap specifies the configuration for the injected sidecar. 
To change how namespaces are selected for injection, you can edit the `MutatingWebhookConfiguration` with...

**Comment by rcaballeromx** [comment]
@ostromart I applied your suggestion thanks.

File: content/docs/setup/kubernetes/additional-setup/sidecar-injection/index.md [hunk 2]
```diff
@@ -160,20 +160,31 @@ sleep-776b7bcdcd-gmvnr   1/1       Running       0          2s
...
-istio-sidecar-injector`.
+When Kubernetes invokes the webhook, the [admissionregistration.k8s.io/v1beta1#MutatingWebhookConfiguration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#mutatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io)
+configuration is applied. The default configuration injects the sidecar into
+pods in any namespace with the `istio-injection=enabled label`. The
+`istio-sidecar-injector` configuration map specifies the configuration for the
+injected sidecar. To change how namespaces are selected for injection, you can
+edit the `MutatingWebhookConfiguration` with the following command:
+
+{{< text bash >}}
+
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/additional-setup/sidecar-injection/index.md]: Please remove the empty lines in the text blocks.

File: content/docs/setup/kubernetes/additional-setup/sidecar-injection/index.md [hunk 3]
```diff
@@ -160,20 +160,31 @@ sleep-776b7bcdcd-gmvnr   1/1       Running       0          2s
...
+$ kubectl edit mutatingwebhookconfiguration istio-sidecar-injector
+
+{{< /text >}}
 
 {{< warning >}}
-The sidecar injector pod(s) should be restarted after modifying the `mutatingwebhookconfiguration`.
+
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/additional-setup/sidecar-injection/index.md]: Also remove empty lines in warning blocks.

**Comment by rcaballeromx** [comment]
Thanks @geeknoid 
I've removed the blank lines.

**Pull request merged**